iAPS, Loop, & AAPS Migration

tmhastings opened the pull request:
- Adds an AAPS Migration page
- Adds a Loop Migration page
- Completes the iAPS Migration page
- Updates mkdocs.yml and relevant index page

tmhastings committed:
Adds AAPS Migration, Completes iAPS Migration

tmhastings committed:
Merge branch 'nightscout:dev' into migration

tmhastings committed:
Update loop-migration.md

tmhastings committed:
Merge branch 'migration' of https://github.com/tmhastings/trio-docs into migration

tmhastings committed:
Update Loop Migration & Remove edit markers

tmhastings committed:
Update formatting of migration pages

tmhastings committed:
Minor formatting fixes and comment out 0.7 warning in 0.2 migration

tmhastings committed:
Formatting changes, images, dial>picker

- Adds Images for Trio 0.2 migration changes explanations
- Replaces "dial" with "picker"
- Minor Formatting fixes
- Rephrasing of Autotune Omission explanation
- Reformats Migration Guide Index page to match other index pages in the docs

tmhastings committed:
Merge branch 'nightscout:dev' into migration

t1dude commented:
AAPS Settings that _do_ exist (table needs to be updated):

1. Max COB: In AAPS, this is located under Preferences - Treatments safety - Max allowed carbs
2. Half basal exercise target: Overview Preferences - Default Temp-targets - Activity target value
3. Display meal presets: Overview Preferences - QuickWizard Settings - Add / edit QuickWizard

We might want to add that Trio does not offer custom automations. One custom automation that most AAPS users add, is automating a hypo temp target when glucose drops under a certain value. When using Trio, this can be achieved by enabling "sensitivity raises target" and/or by adding a hypo temp target or a hypo override that is easily accessible for manual use.

tmhastings committed:
Adding suggestions from @t1dude

tmhastings committed:
Update loop-migration.md

Add instructions for updating nightscout plugins fro Trio use based on recomendations by @MikePlante1

tmhastings commented:
Updates complete. Please review again and let me know if anything needs correcting. Thank you!

t1dude reviewed (approved):
Reviewed and approved :)
